strings/strings.c

```diff
@@ -9,7 +9,9 @@
 */
 int string_length(char s[])
 {
-
+    int len;
+    len = sizeof(s);
+    return len;
 }
```

Comment:
This is not returning the correct result. I believe that when you use sizeof this way, you are going to return the amount of space the string takes up in memory and not the actual number of letters contained in the string.